src/JSInterop/Microsoft.JSInterop/src/Infrastructure/DotNetDispatcher.cs

```diff
@@ -371,7 +374,8 @@ private static (MethodInfo methodInfo, Type[] parameterTypes) GetCachedMethodInf
                         $"the [JSInvokable] attribute.");
                 }
 
-                result.Add(identifier, (method, parameterTypes));
+                var invoker = ObjectMethodExecutor.GetExecutor(method, default);
```

Comment:
We need to be really careful with this. It's likely for WASM that using reflection is the implementation we want due to size.

Reply:
Indeed. The numbers suggest < 20% speed/memory differences, so I wouldn't say this stands out as a good tradeoff.